### PR TITLE
Complete number_of_seconds homework

### PR DESCRIPTION
**Pull request by timsjpark**
[#104778992]

### CODE REVIEW

**Comment by jasonnoble** [comment]
Failed travis tests... needs rebasing.

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+seconds_per_minute = 60
+minutes_per_hour = 60
+hours_per_day = 24
+days_per_week = 7
+days_per_year = 365
+seconds_per_hour = seconds_per_minute * minutes_per_hour
+seconds_per_day = seconds_per_hour * hours_per_day
+seconds_per_year = seconds_per_day * days_per_year
+
+puts 'There are 60 seconds in a minute'
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: You have a seconds_per_minute variable, you should use it.

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+puts 'There are 60 minutes in an hour'
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: same...  And the next two lines.

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+puts 'There are 24 hours in a day'
+puts 'There are 7 days in a week'
+puts 'That means there are:'
+puts "  #{seconds_per_hour} seconds in an hour,"
+puts "  #{seconds_per_day} seconds in a day,"
+puts "  #{seconds_per_day * days_per_week} seconds in a week"
+puts "That means when you turn 20, you've been alive for " +
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: Is there a way to output a line of test without a new line occurring?

File: numbers_letters_and_variables/number_of_seconds.rb (hunk 1 above)
Comment (timsjpark) [inline numbers_letters_and_variables/number_of_seconds.rb]: I use seconds_per_minute to calculate seconds_per_hour

File: numbers_letters_and_variables/number_of_seconds.rb (hunk 2 above)
Comment (timsjpark) [inline numbers_letters_and_variables/number_of_seconds.rb]: I use minutes_per_hour and hours_per_day to calculate seconds_per_hour and seconds_per_day respectively

File: numbers_letters_and_variables/number_of_seconds.rb (hunk 3 above)
Comment (timsjpark) [inline numbers_letters_and_variables/number_of_seconds.rb]: I don't understand the question. 

Do you mean is there another way to concatenate strings other than with the + symbol? In that, I would replace + with the \ to print one line with the puts method. Alternatively you could use the
print method in conjunction with puts, but that looks less elegant, IMO.

File: numbers_letters_and_variables/number_of_seconds.rb (hunk 1 above)
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: Right... You should use interpolation on lines 12-15.

File: numbers_letters_and_variables/number_of_seconds.rb (hunk 3 above)
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: `puts` is one way to output a line of text followed by a new line.  Is there another method that `print`s the line without adding a new line?

i.e. `print` the first line, `puts` the second line.

**Pull request merged**